Annex60/Experimental/Benchmarks/AirFlow/Examples/MultipleFloorsVectors.mo

```diff
@@ -0,0 +1,75 @@
+within Annex60.Experimental.Benchmarks.AirFlow.Examples;
+model MultipleFloorsVectors
+  "Test case for air flow between multiple floors in vector design"
+  extends Modelica.Icons.Example;
+
+  replaceable package Medium = Annex60.Media.Air;
+
+  parameter Integer nZones(min=1) = 4 "Number of zone elements per floor";
+  parameter Integer nFloors(min=2) = 2 "Number of floors";
+
+  parameter Modelica.SIunits.Temperature TRoom = 298.15
+    "Indoor air temperature of room in K";
+  parameter Modelica.SIunits.Temperature THallway = 293.15
+    "Indoor air temperature of hallway in K";
+  parameter Modelica.SIunits.Temperature TStaircase = 293.15
+    "Indoor air temperature of staircase in K";
+  parameter Modelica.SIunits.Height heightRooms = 3 "Height of rooms in m";
+
+  parameter Modelica.SIunits.Length lengthZone = 5 "Length of room in m";
+  parameter Modelica.SIunits.Length widthZone = 5 "Width of room in m";
+  parameter Modelica.SIunits.Length widthHallway = 3 "Width of room in m";
+  parameter Real doorOpening = 1
+    "Opening of door (between 0:closed and 1:open)";
+
+  parameter Boolean forceErrorControlOnFlow = true
+    "Flag to force error control on m_flow. Set to true if interested in flow rate";
+
+  inner Modelica.Fluid.System system
+    annotation (Placement(transformation(extent={{80,80},{100,100}})));
+  Components.Floor floor[nFloors](
+    each nZones=nZones,
+    each TRoom=TRoom,
+    each THallway=THallway,
+    each TStaircase=TStaircase,
+    each heightRooms=heightRooms,
+    each lengthZone=lengthZone,
+    each widthZone=widthZone,
+    each widthHallway=widthHallway,
+    redeclare each package Medium = Medium,
+    each forceErrorControlOnFlow=forceErrorControlOnFlow)
```

Comment:
Please add a comment string to all instances as they will show in plot windows and in the schematic diagram editor.